script/deploy/mainnet/007_UpgradeLidoARMMorphoScript.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.23;
+
+// Foundry imports
+import {console} from "forge-std/console.sol";
+
+// Contract imports
+import {LidoARM} from "contracts/LidoARM.sol";
+import {Mainnet} from "contracts/utils/Addresses.sol";
+import {MorphoMarket} from "contracts/markets/MorphoMarket.sol";
+
+// Deployment imports
+import {GovProposal, GovSixHelper} from "contracts/utils/GovSixHelper.sol";
+import {AbstractDeployScript} from "../AbstractDeployScript.sol";
+
+contract UpgradeLidoARMMorphoScript is AbstractDeployScript {
+    using GovSixHelper for GovProposal;
+
+    GovProposal public govProposal;
+
+    string public constant override DEPLOY_NAME = "007_UpgradeLidoARMMorphoScript";
+    bool public constant override proposalExecuted = false;
+
+    LidoARM lidoARMImpl;
+    MorphoMarket morphoMarket;
+
+    function _execute() internal override {
+        console.log("Deploy:", DEPLOY_NAME);
+        console.log("------------");
+
+        // 1. Deploy new Lido implementation
+        uint256 claimDelay = tenderlyTestnet ? 1 minutes : 10 minutes;
+        lidoARMImpl = new LidoARM(Mainnet.STETH, Mainnet.WETH, Mainnet.LIDO_WITHDRAWAL, claimDelay);
```

Comment:
I think the we need to set a  _allocateThreshold and possibly _minSharesToRedeem on the LidoARM internal constructor parameters. I think these were set to 0, 0 but I'm guessing these should be an actual value.